### DISCUSSION TITLE
eslint update 9.x.x

### DISCUSSION THREAD

**Discussion by tinohager** [quasarframework/quasar#17378] [General - Components / Directives / etc]
It is possible to update the eslint version?

> npm WARN deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported

https://npmgraph.js.org/?q=quasar#deps=devDependencies

![image](https://github.com/user-attachments/assets/36d90bb2-4c88-48eb-97a0-9e3b5c711dc6)

**Comment by AmosSzust** [comment]
+1

> npm error code ERESOLVE
> npm error ERESOLVE could not resolve
> npm error
> npm error While resolving: @quasar/app-vite@1.9.4
> npm error Found: eslint@9.8.0
> npm error node_modules/eslint
> npm error   dev eslint@"^9.8.0" from the root project
> npm error   peer eslint@"^8.57.0 || ^9.0.0" from @typescript-eslint/eslint-plugin@8.0.1
> npm error   node_modules/@typescript-eslint/eslint-plugin
> npm error     dev @typescript-eslint/eslint-plugin@"^8.0.1" from the root project
> npm error   3 more (@typescript-eslint/parser, eslint-config-prettier, eslint-plugin-vue)
> npm error
> npm error Could not resolve dependency:
> npm error peerOptional eslint@"^8.11.0" from @quasar/app-vite@1.9.4
> npm error node_modules/@quasar/app-vite
> npm error   dev @quasar/app-vite@"^1.9.4" from the root project
> npm error
> npm error Conflicting peer dependency: eslint@8.57.0
> npm error node_modules/eslint
> npm error   peerOptional eslint@"^8.11.0" from @quasar/app-vite@1.9.4
> npm error   node_modules/@quasar/app-vite
> npm error     dev @quasar/app-vite@"^1.9.4" from the root project
> npm error
> npm error Fix the upstream dependency conflict, or retry
> npm error this command with --force or --legacy-peer-deps
> npm error to accept an incorrect (and potentially broken) dependency resolution.
